core/templates/pages/blog-dashboard-page/blog-dashboard-tile/blog-dashboard-tile.component.spec.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
```suggestion
// Copyright 2021 The Oppia Authors. All Rights Reserved.
```
Please check elsewhere too.

Reply:
Done.